package.json

```diff
@@ -24,6 +24,7 @@
     "if-env": "^1.0.4",
     "import-sort-style-eslint": "^6.0.0",
     "lerna": "^2.11.0",
+    "node-addon-api": "3.1.0",
```

Comment:
Why do we need a direct dependency on `node-addon-api` there?
I think it's redundant.

Reply:
https://issues.redhat.com/browse/CRW-2156?focusedCommentId=18821361&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-18821361